torch/_dynamo/output_graph.py

```diff
@@ -2229,6 +2197,106 @@ def maybe_lift_tracked_freevar_to_input(self, arg):
             return arg
         return self.lift_tracked_freevar_to_input(arg)
 
+    def maybe_bind_symbols(self, example_value, src: Optional[Source]):
```

Comment:
Previously we had `add_symbol_bindings` with helper functions `bind_symint` and `handle_tensor`. Now they're kind of flattened out. The structure made some sense to me (add_symbol_bindings is the main API, and it does what it says it does).

Now, we have maybe_bind_symbols, handle_tensor, and bind_symint. I don't know what these do, or if they're supposed to be called separately, etc. Can you explain a bit more?

If these functions are used to handle adding symints to graph inputs... then let's make the naming clearer

Reply:
Okay I think maybe_bind_symbols is the entrypoint. Can we call it something like... "add_symbol_bindings"? (i'm not sure why the "maybe" is there, because we don't do this in export?)

Reply:
yeah, also it's because not all example_value have symbols in them.

Reply:
You can name this whatever you want, I would put a comment here saying what this does (given a Tensor, it looks at all of the symbols and adds them to be inputs of the current SubgraphTracer)